### ISSUE TITLE
is it possible to change the credits?

### ISSUE THREAD

**Issue by germ12** [ShadowMario/FNF-PsychEngine#507]
### What is your question?

i wanted to know so i can add myself into the credits

**Comment by FuseIsHere813** [comment]
Yes, you need to edit **CreditsState.hx** including your own credit icon, twitter link (optional, you can use YouTube as an alternative.
